webapp/src/config.js

```diff
@@ -0,0 +1,3 @@
+const ENDPOINT = process.env.REACT_APP_ENDPOINT;
+
```

Comment:
Without a valid configuration for the endpoint (missing port), the app will throw Unhandled Rejection (TypeError): action.payload is undefined

We should catch this behavior and display that something is wrong.

Reply:
Where would you want to add error checking code, and would it output to the console or to the browser?

IMO, there's not much of a point to error checking env vars since it's easy for the developer to test if it working and users will never be affected.

We could display an error if any `redux-api-middleware` actions fail.

Reply:
Kind of just thinking console, I was just surprised it didn't trigger GET_COURSES_ERROR from the RSAA.

Reply:
Were you thinking something like f856a6c?

The error reducer correctly triggered for me when I set `REACT_APP_ENDPOINT=http://localhost`. 🤷